0-bootstrap/variables.tf

```diff
@@ -30,6 +30,24 @@ variable "default_region" {
   default     = "us-central1"
 }
 
+variable "default_region_2" {
+  description = "Secondary default region to create resources where applicable."
+  type        = string
+  default     = "us-west1"
+}
+
+variable "default_region_gcs" {
+  description = "Case-Sensitive default region to create gcs resources where applicable."
+  type        = string
+  default     = "US"
+}
+
+variable "default_region_kms" {
+  description = "Secondary default region to create kms resources where applicable."
+  type        = string
+  default     = "us"
+}
```

Comment:
in the helper

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/helpers/foundation-deployer/global.tfvars.example#L139

the values for the variables in step 4 need to be updated  and a note added that they can be used to override the default values from step 0

Reply:
Thanks! Fixed in 941c155bf4a837d875466a0a92d7ac12237ac5b3